src/components/PageTitle/PageTitle.tsx

```diff
@@ -60,7 +62,11 @@ export default function PageTitle({
       style={{
         height: `${pageTitleHeight}px`,
         position: isSticky ? "fixed" : "relative",
-        marginTop: !isSticky ? "250px" : `${lastPosition}px`,
+        marginTop: !isSticky
+          ? windowHeight >= 850
+            ? "350px"
+            : "250px"
+          : `${lastPosition}px`,
```

Comment:
All this being hardcoded will be a problem when handling different widths with media